content/docs/immutable-updates.md

```diff
@@ -0,0 +1,91 @@
+---
+id: immutable-updates
+title: Immutable Updates
+permalink: docs/immutable-updates.html
+layout: docs
+---
+
+To ensure immutability without performing expensive deep cloning, it is good practice to replace only the values which need updating and to reuse the rest of the original object.
+
+### Updating Object Properties
+
+To update an object property, you can make use of the [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign): 
```

Comment:
We should reorder these sections. Show `Object.assign` first since it's in the standard, then mention a syntax *proposal* (but also mention it's enabled by default in Create React App, and already ships in Node and Chrome).